src/System.Threading/tests/InterlockedTests.netcoreapp.cs

```diff
@@ -97,7 +98,7 @@ private LockCookie EnterSlow()
             }
         }
 
-        [Fact]
+        [ConditionalFact(nameof(PlatformDetection.IsNotArm64Process))] // issue: https://github.com/dotnet/coreclr/issues/20215
```

Comment:
This needs to be: `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process))]`